tools/dotnet-msidentity/dotnet-msidentity.csproj

```diff
@@ -26,7 +26,7 @@
     <Product>Microsoft identity platform app registration tool</Product>
     <Description>
       This package is a dotnet global tool which registers new Azure AD or Azure AD B2C applications, and updates your code accordingly, or modifies/edits existing
```

Comment:
We should mention somewhere in here that this tool only work with asp.net core projects.

Reply:
added "Works with ASP.NET Core 3.1 and .NET 5.0, 6.0 projects."

Reply:
Ok that is good, I may change it later when I do the overall review of the content.